radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -233,6 +233,35 @@ elseif(PCB STREQUAL X7)
     add_definitions(-DRADIO_T8)
     add_definitions(-DEEPROM_VARIANT=0x4004)
     add_definitions(-DMANUFACTURER_RADIOMASTER)
+    elseif(PCBREV STREQUAL COMMANDO8)
+    set(DEFAULT_INTERNAL_MODULE MULTIMODULE CACHE STRING "Default internal module")
+    set(INTERNAL_MODULE_SERIAL YES)
+    set(HARDWARE_EXTERNAL_MODULE YES)
+    set(FLAVOUR commando8)
+    set(NAVIGATION_TYPE x7)
+    set(CPU_TYPE STM32F4)
+    set(CPU_TYPE_FULL STM32F407xE)  # for size report
+    add_definitions(-DSTM32F407xx)
+    set(LINKER_SCRIPT targets/taranis/stm32f4_flash.ld)
+    set(FIRMWARE_TARGET_SRC
+      ${FIRMWARE_TARGET_SRC}
+      startup_stm32f40_41xxx.s
+      ../../thirdparty/STM32F4xx_DSP_StdPeriph_Lib_V1.8.0/Libraries/STM32F4xx_StdPeriph_Driver/src/stm32f4xx_flash.c
+      )
+    add_definitions(-DSTM32F407xx -DSTM32F40_41xxx)  
+    set(ROTARY_ENCODER NO)
+    set(LUA_EXPORT lua_export_commando8)
+    set(BLUETOOTH NO)
+    set(UNRTCLOCK YES)
```

Comment:
It is probably preferable to have a parameter named `USE_RTC_CLOCK` defined at global level of this file, and force its value here.